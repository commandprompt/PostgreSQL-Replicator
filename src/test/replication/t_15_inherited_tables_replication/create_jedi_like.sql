CREATE TABLE jedi(LIKE rebels, force INTEGER);
ALTER TABLE jedi ADD PRIMARY KEY(id);
