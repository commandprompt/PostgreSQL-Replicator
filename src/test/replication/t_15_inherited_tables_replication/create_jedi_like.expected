psql:.*:2: NOTICE:  ALTER TABLE / ADD PRIMARY KEY will create implicit index "jedi_pkey" for table "jedi"
